### PR TITLE
add application doc(en)

### PR DESCRIPTION
**Pull request by littletomatodonkey**
See title.

### CODE REVIEW

File: docs/en/application/object_detection_en.md [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+| +`fpn channel=64` | 640x640 | 640x640 | 35.1% | 67.450 |
+| +`ssld pretrain` | 640x640 | 640x640 | 36.3% | 67.450 |
+| +`ciou loss` | 640x640 | 640x640 | 37.1% | 67.450 |
+| +`DCNv2` | 640x640 | 640x640 | 39.4% | 60.345 |
+| +`3x, multi-scale training` | 640x640 | 640x640 | 41.0% | 60.345 |
+| +`auto augment` | 640x640 | 640x640 | 41.4% | 60.345 |
+| +`libra sampling` | 640x640 | 640x640 | 41.6% | 60.345 |
+
+
+Based on the ablation experiments, Cascade RCNN and larger inference scale(1000x1500) are used for better performance. The final COCO mAP is 47.8% when the
```
Comment (shippingwang) [inline docs/en/application/object_detection_en.md:25]: delete ''when the'' ?

File: docs/en/application/transfer_learning_en.md [hunk 2]
```diff
@@ -0,0 +1,88 @@
+# Transfer learning in image classification
+
+Transfer learning is an important part machine learning, which is widely used in various fields such as text and images. Here we mainly introduce transfer learning in the field of image classification, which is often called domain transfer, such as migration of the ImageNet classification model to the specified image classification task, such as flower classification.
```
Comment (shippingwang) [inline docs/en/application/transfer_learning_en.md:3]: part of

File: docs/en/application/transfer_learning_en.md [hunk 3]
```diff
@@ -0,0 +1,88 @@
...
+
+### 1.1 Grid search
+
+For grid search, which is also called exhaustive search, the optimal value is determined by finding the best solution from all solutions in the search space. The method is simple and effective, but when the search space is large, it takes huge computing resource.
+
+### 1.2 Bayesian search
+
+Bayesian search, which is also called Bayesian optimization, is realized by randomly selecting a group of hyperparameters in the search space. Gaussian process is used to update the hyperparameters, compute their expected mean and variance according to the performance of the previous hyperparameters. The larger the expected mean, the greater the probability of being close to the optimal solution. The larger the expected variance, the greater the uncertainty. Usually, the hyperparameter point with large expected mean is called `exporitation`, and the hyperparameter point with large variance is called `exploration`. Acquisition function is defined to balance the expected mean and variance. The currently selected hyperparameter point is viewed as the optimal position with maximum probability.
```
Comment (shippingwang) [inline docs/en/application/transfer_learning_en.md:15]: exporitation -> exportation?

File: docs/en/application/transfer_learning_en.md [hunk 4]
```diff
@@ -0,0 +1,88 @@
...
+lr: [0.1, 0.03, 0.01, 0.003, 0.001, 0.0003, 0.0001]
+
+l2 decay: [1e-3, 3e-4, 1e-4, 3e-5, 1e-5, 3e-6, 1e-6]
+
+label smoothing: [False, True]
+
+mixup: [False, True]
+```
+
+It takes 196 times for grid search, and takes 10 times less for Bayesian search. The baseline is trainin using ImageNet1k pretrained model based on ResNet50_vd and fixed scheme. The follow shows the experiments.
```
Comment (shippingwang) [inline docs/en/application/transfer_learning_en.md:40]: trainin -> training
or
is trained by using

File: docs/en/application/transfer_learning_en.md [hunk 5]
```diff
@@ -0,0 +1,88 @@
...
+| Food101            | 87.07%   | 87.52%   | 196 |  87.33%     | 23         |
+| SUN397             | 63.27%   | 64.84%   | 196 |  64.55%     | 20         |
+| Caltech101         | 91.71%   | 92.54%   | 196 |  92.16%     | 14         |
+| DTD                | 76.87%   | 77.53%   | 196 |  77.47%     | 13         |
+| Stanford Cars      | 85.14%   | 92.72%   | 196 |  92.72%     | 25         |
+| FGVC Aircraft      | 80.32%   | 88.45%   | 196 |  88.36%     | 20         |
+
+
+- The above experiments verify that Bayesian search only reduces the accuracy by 0% to 0.4% under the condition of reducing the number of searches by about 10 times compared to grid search.
+- Thes earch space can be expaned easily using Bayesian search.
```
Comment (shippingwang) [inline docs/en/application/transfer_learning_en.md:56]: The search

File: docs/en/application/transfer_learning_en.md [hunk 6]
```diff
@@ -0,0 +1,88 @@
...
+| Leaves     | class:6<br />train:5256<br />valid:2278   | 0.8212/0.8482                                            | 0.8385/0.8659                                             |
+| Container vehicle       | Class:115<br />train:4879<br />valid:2094 | 0.6230/0.9556                                            | 0.9524/0.9702                                             |
+| Chair         | class:5<br />train:169<br />valid:78      | 0.8557/0.9688                                            | 0.9077/0.9792                                             |
+| Geology         | class:4<br />train:671<br />valid:296     | 0.5719/0.8094                                            | 0.6781/0.8219                                             |
+
+- The above experiments verified that for fixed parameters, compared with the pretrained model on ImageNet, using the large-scale classification model as a pretrained model can help us improve the the model performance on a new dataset in most cases. Parameter search can be further helpful to the model performance.
```
Comment (shippingwang) [inline docs/en/application/transfer_learning_en.md:82]: improve the the -> improve the

File: docs/en/application/object_detection_en.md (hunk 1 above)
Comment (littletomatodonkey) [reply docs/en/application/object_detection_en.md]: fixed

File: docs/en/application/transfer_learning_en.md (hunk 2 above)
Comment (littletomatodonkey) [reply docs/en/application/transfer_learning_en.md]: fixed

File: docs/en/application/transfer_learning_en.md (hunk 3 above)
Comment (littletomatodonkey) [reply docs/en/application/transfer_learning_en.md:15]: That is not typo

File: docs/en/application/transfer_learning_en.md (hunk 4 above)
Comment (littletomatodonkey) [reply docs/en/application/transfer_learning_en.md]: fixed

File: docs/en/application/transfer_learning_en.md (hunk 5 above)
Comment (littletomatodonkey) [reply docs/en/application/transfer_learning_en.md]: fixed

File: docs/en/application/transfer_learning_en.md (hunk 6 above)
Comment (littletomatodonkey) [reply docs/en/application/transfer_learning_en.md]: fixed

**Pull request merged**